src/styles/styled.js

```diff
@@ -35,6 +35,8 @@ export default function styledWrapper(...args: any) {
   // Ex: <StyledDiv $style={{color: 'red'}} />
   // Issue for supporting this natively in styletron:
   // https://github.com/rtsao/styletron/issues/221
+
+  // $FlowFixMe
```

Comment:
The `$FlowFixMe` isn't preferable here, but with the complexity of the nested structure I wasn't able to correct the typings here.